frontend/e2e/page-objects/InquiryPage.ts

```diff
@@ -0,0 +1,73 @@
+import { expect } from "@playwright/test"
+import HelperBase from "./helperBase"
+
+export default class InquiryPage extends HelperBase {
+  async navigateToInquiriesPage() {
+    await this.page.waitForSelector("p.chakra-text.css-0", { timeout: 5000 })
+    await expect(this.page.locator("p.chakra-text.css-0")).toHaveText(
+      "Welcome back, nice to see you again!",
+    )
+    await this.page.getByText("Inquiries").click()
+  }
+
+  async openInquiryForm() {
+    const inquiryTitle = this.page.getByRole("heading", {
+      name: "Inquiries Management",
+    })
+    await expect(inquiryTitle).toHaveText("Inquiries Management")
+    await this.page.getByRole("button", { name: "Inquiry" }).click()
+    await this.page.waitForSelector(".chakra-modal__footer", {
+      state: "visible",
+    })
+    await expect(
+      this.page.getByLabel("Add Inquiry").getByText("Add Inquiry"),
+    ).toHaveText("Add Inquiry")
+  }
+
+  async submitInquiry() {
+    const randomQuestion = this.getRandomQuestion()
```

Comment:
As a general principle, it's not a good idea to have randomness in tests. The reason is that if some random value causes the test to fail, simply retrying the failed test can cause it to succeed, leading the author to think the test was simply flaky. I would recommend picking specific values, or maybe having multiple tests with multiple edge-case values.

Reply:
Thank you for your feedback! I understand the concern about having randomness in tests and how that can lead to flaky results. To address this, I’ve made changes to ensure that each test uses a fixed input value. 
I’ve removed the random selection of questions and replaced it with predefined inquiry text for each test. Each test case now independently verifies that an inquiry is added and displayed in the list. I hope this resolves the concern and ensures better stability for the tests.